### ISSUE TITLE
Issue list with Favlist add-on

### ISSUE THREAD

**Issue by pooja-muchandikar** [rtMediaWP/rtMedia#1523]
- [x] 1. Edit favlist - icon and titles are not aligned
![image](https://user-images.githubusercontent.com/41000648/71656186-5b4b6700-2d60-11ea-9d21-290b704e4892.png)

- [x] 2.  2. Edit favlist - title text field is smaller, if the title is lengthy its not completely visible. Can we make the title field little bigger?
![image](https://user-images.githubusercontent.com/41000648/71656167-4a025a80-2d60-11ea-9e09-fce047ea6390.png)

**Comment by dishitpala** [comment]
@pooja-muchandikar 
>2. 2. Edit favlist - title text field is smaller, if the title is lengthy its not completely visible. Can we make the title field little bigger?

I don't think it's an issue because we don't have any fixed char. length for an album name 
So suppose we give `50px` to the text field and add title as `rtparty goa 17 January 2020` then it's gonna be the same issue 

@thrijith please share your thoughts on this.

**Comment by thrijith** [comment]
@dishitpala instead of fixed width try giving `100%` so it will take space based on the parent and should work in other resolutions as well.

**Comment by dishitpala** [comment]
@pooja-muchandikar 
The issue list in the description is fixed and merged in `pre-release` branch 
Please do the needful.

**Comment by pooja-muchandikar** [comment]
@dishitpala 
Tested with `pre-release` branch 
>  1. Edit favlist - icon and titles are not aligned
- Looks fine :heavy_check_mark: 

> 2. Edit favlist - title text field is smaller, if the title is lengthy its not completely visible. Can we make the title field little bigger?
- Looks fine :heavy_check_mark: 

```
Test Environment
WordPress: v5.3.2
BuddyPress: v5.1.2
rtMedia: v4.5.11 pre-release branch
Theme: 2019, 2020
Browser: Chrome, Firefox and Safari
```

**Comment by pooja-muchandikar** [comment]
Tested with `develop` branch and the UI fix added looks fine
```
Test Environment
WordPress: v5.3.2
BuddyPress: v5.1.2
rtMedia: v4.5.11 develop branch
Theme: 2019, 2020
Browser: Chrome, Firefox and Safari 
```

**Comment by pooja-muchandikar** [comment]
Tested with `master` branch and the fix added works as expected.

**Comment by pooja-muchandikar** [comment]
Released with `v4.6.0`
